src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx

```diff
@@ -23,37 +19,34 @@ Complete the steps below to collect Kafka metrics from Confluent.
 
 ## Step 2: Prerequisites [#prerequisites]
 
-* Ensure [Go](https://go.dev/doc/install) is installed before proceeding.
-* Set [GOPATH](https://go.dev/doc/gopath_code) to `$GOPATH/bin` and add to the PATH variable.
-
-## Step 3: Compile from PR source repo [#compile]
+* You must have a docker daemon running
+* You must have [Docker Compose](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/confluentcloud) installed
+* You will need to generate a [TLS authentication key](https://docs.confluent.io/platform/current/kafka/encryption.html) from your [Confluent Cloud account](https://www.confluent.io/get-started/).  
 
-<Callout variant="important">
-  New Relic supports the OpenTelemetry community by contributing our work upstream to both the [Core](https://github.com/open-telemetry/opentelemetry-collector) and [Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) repos.
 
-  When [PR22839](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/22839) on the [OpenTelemetry Collector Contrib repo](https://github.com/open-telemetry/opentelemetry-collector-contrib) has been merged, the documentation below will be updated to reflect the main branch of the Contrib repo.
-</Callout>
+## Step 3: Download or clone the examples repo [#example]
 
-See [https://github.com/4demos/opentelemetry-collector-contrib.git](https://github.com/4demos/opentelemetry-collector-contrib) for latest installation instructions.
+This setup uses the example collector configuration in [New Relic's OpenTelemetry Examples repo](https://github.com/newrelic/newrelic-opentelemetry-examples).
 
-```bash
-git clone https://github.com/4demos/opentelemetry-collector-contrib.git
-cd opentelemetry-collector-contrib
-make otelcontribcol
-
-```
+Download the repo above, and navigate to the [Confluent Cloud example](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/confluentcloud) directory. For more information, you can check the `README` there as well.
 
-The binary will be installed under `./bin`
+## Step 4: Add the pem key files and set variables [#configure-opentelemetry-collectors]
 
-## Step 4: Configure Opentelemetry collectors [#configure-opentelemetry-collectors]
+* Add the TLS authentication keys (key.pem, cert.pem, and ca.pem files) created in the pre-requisites into the `confluentcloud` directory.
```

Comment:
This step is a little vague and I think we can add more content to make it clearer. In the README, it links out to this doc: https://docs.confluent.io/platform/current/kafka/encryption.html

It'd be best if we explained or distilled what steps they need to take to create these authentication keys. If we link out to the repo, users may navigate away and not finish this process. Best to deliver all relevant action items in one spot as best we can.